app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -390,4 +303,76 @@ class ChatSettingsFragment : PreferenceFragmentCompat(), ISettingsView {
         super.onDestroyView()
         settingsPresenter.onDetach()
     }
+
+    override fun setLoginPreferences(message: String, loginMessage: String) {
+        loginLogout.title = loginMessage
+        displayEmail.title = message
+
+        /*   set Mic preferences    */
```

Comment:
A single line comment.
```suggestion
        // Set mic preferences
```